EIPS/eip-2315.md

```diff
@@ -59,7 +61,9 @@ _Note 3: The description above lays out the semantics of this feature in terms o
 
 ## Rationale
 
-This is almost the smallest change that provides native subroutines without breaking backwards compatibility.
+We modeled this design on the, simple, proven, [archetypal Forth virtual machine](https://users.ece.cmu.edu/~koopman/stack_computers/index.html) of [1970](http://www.ultratechnology.com/4th_1970.pdf).  It is a two-stack design -- the data stack is supplemented with a return stack to support jumping into and returning from subroutines, as specified above.  Most other models place the return address on the stack.   The separate return stack ensures that the return address cannot be overwritten or mislaid, and obviates any need to swap the return address past the arguments on the stack.
```

Comment:
As mentioned previously, external links should be avoided.  For the second link you can just download a copy of the PDF and put it in `../assets/eip-2315/4th_1970.pdf` and link to it.

Alternatively, you could just leave both references out since I don't think they are necessary for a technical document since "4th does it" is an appeal to authority, and not a sound argument.

Reply:
@MicahZoltu I'm not appealing to authority -- I'm describing what choice I made and why.  The Forth model really is simple and proven by decades of experience.  Also archetypal: it's pretty much the canonical stack machine.  The links are references to back up those claims.  Otherwise it seems I'm appealing to my own authority (which I do too often as it is. ;)

Unfortunately, the most useful reference is to Koopman, which copyright forbids us to share on our server.

Reply:
@MicahZoltu Links removed. They'll either have to do their homework or take my word for it.